yoti_python_sdk/doc_scan/session/create/notification_config.py

```diff
@@ -16,7 +16,7 @@ class NotificationConfig(YotiSerializable):
     to poll for the state of the Session.
     """
 
-    def __init__(self, auth_token, endpoint, topics=None):
+    def __init__(self, auth_token, endpoint, auth_type, topics=None):
```

Comment:
I know we have builders and so we're not recommending clients initialise with this, but at all times when adding to functions we try to:
1) put new parameters at the end of the parameter list. If it's as it is currently, then any existing calls to this function which are passing `topics` will now pass this as the `auth_type`
2) We initialise as `None`, so that clients aren't _required_ to set this parameter